compiler/vm.nim

```diff
@@ -2185,29 +2185,36 @@ proc evalStaticStmt*(module: PSym; g: ModuleGraph; e: PNode, prc: PSym) =
 proc setupCompileTimeVar*(module: PSym; g: ModuleGraph; n: PNode) =
   discard evalConstExprAux(module, g, nil, n, emStaticStmt)
 
-proc prepareVMValue(arg: PNode): PNode =
+proc prepareVMValue(g: ModuleGraph, arg: PNode): PNode =
   ## strip nkExprColonExpr from tuple values recurively. That is how
```

Comment:
pre-existing but s/recurively/recursively

Reply:
it's ok to leave this as future work but then please don't mark this as resolved otherwise this is forgotten

Reply:
ah i see 1a407402a4614fc9100617f71c2d3c9b8a8367a5, thanks!

Reply:
maybe make a small PR to clean them
https://github.com/nim-lang/Nim/commit/1a407402a4614fc9100617f71c2d3c9b8a8367a5